seed.js

```diff
@@ -268,6 +268,62 @@ const seed = async () => {
     })
   }).flat())
 
+  const [ zoe, zaina, adam, jamil ] = await Promise.all([
```

Comment:
Oops this was supposed to be in my other branch, but here we go I seeded some users

Reply:
I wish I had adam@adam.com